src/main/java/org/grpc/service/repositories/EmployeeRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.grpc.service.repositories;
+import org.grpc.service.entities.Employee;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface EmployeeRepository extends CrudRepository<Employee, Long>{
```

Comment:
What is CrudRepository and how to use it?

Reply:
CrudRepository is a simple JPA class off of which you can expand do write your own repository. CRUD as in Create, Read, Update and Delete, and then you write just in plain words what do you want it to do. It automatically generates the code, in our case, SQL